src/app/apps/vacunacion/services/inscripcion.service.ts

```diff
@@ -10,20 +10,12 @@ export class InscripcionService {
 
     constructor(private server: Server) { }
 
-    // get(params: any): Observable<any[]> {
-    //     return this.server.get(this.reglaUrl, { params: params, showError: true });
-    // }
-
-    // getById(id: String): Observable<any> {
-    //     return this.server.get(this.reglaUrl + '/' + id, null);
-    // }
+    search(params): Observable<any> {
+        return this.server.get(`${this.inscripcionUrl}/consultas`, { params });
```

Comment:
```suggestion
        return this.server.get(`${this.inscripcionUrl}/consultas`, { params, showErrors: false });
```
Así se puede evitar que muestre el `plex.info()` que es medio invasivo